### PR TITLE
Fixed: Result point evaluation for multi-patch models

### PR DESCRIPTION
**Pull request by kmokstad**
This (the SIMoutput fix) is needed for the digital-twin model, for instance, when there are more than one result points and not all one the same patch. The other commit is more cosmetic, but enables the possibility run without the -controller also (for debugging, etc.)

### CODE REVIEW

**Comment by kmokstad** [comment]
jenkins build this with downstreams please

File: src/IFEM.C [hunk 1]
```diff
@@ -141,15 +150,48 @@ int IFEM::Init (int arg_c, char** arg_v, const char* title)
...
 
   std::cout << std::endl;
 
   return 0;
 }
 
 
+void IFEM::Close ()
+{
+  if (fifo) delete fifo;
```
Comment (akva2) [inline src/IFEM.C]: delete is null safe.

**Comment by kmokstad** [comment]
The thought was that it in most cases the FIFO object is not used (only when run with -controller), and then it is more plausible to allocate it dynamically when needed, instead of an always present static member. And I wanted an easy easy to detect if run with controller not not, without needing to check the command-line arguments. Then we don't need the `enableController` member in SIMoptions also.

**Comment by kmokstad** [comment]
jenkins build this with downstreams please

**Pull request merged**